docs/how-to-guides/advanced-user-guides/istio-setup-guide/istio-setup-guide.md

```diff
@@ -6,6 +6,12 @@ title: Istio Setup Guides
   <link rel="canonical" href="https://ranchermanager.docs.rancher.com/how-to-guides/advanced-user-guides/istio-setup-guide"/>
 </head>
 
+:::note
+
+[Rancher-Istio](https://github.com/rancher/istio-installer) will be deprecated in upcoming Rancher releases; turn to the Application Collection build of Istio for enhanced security (included in SUSE Rancher Prime subscriptions).
```

Comment:
Shouldn't we point out to https://github.com/rancher/charts/tree/dev-v2.10/charts/rancher-istio?

Reply:
We can, I just pointed out to the installer because that's what in the approved text